src/components/enterprise-page/EnterprisePage.test.jsx

```diff
@@ -24,47 +27,93 @@ describe('<EnterprisePage />', () => {
   beforeEach(() => {
     jest.clearAllMocks();
   });
+  const queryClient = new QueryClient({
+    queryCache: new QueryCache({
+      onError: queryCacheOnErrorHandler,
+    }),
+  });
+  const queryClientProviderWrapper = ({ children }) => (
```

Comment:
nit: might recommend making it more obvious that this is a React component, not just a function by making the variable name capitalized.

```suggestion
  const QueryClientProviderWrapper = ({ children }) => (
```

And subsequently rendering it like a normal React component vs. calling the function, e.g.:

```jsx
const wrapper = mount(
    <QueryClientProviderWrapper>
      <AppContext.Provider value={{ authenticatedUser: { ...mockUser } }}>
        <EnterprisePage>
          <div className="did-i-render" />
        </EnterprisePage>
      </AppContext.Provider>,
    </QueryClientProviderWrapper>
);
```

Taking this further, would it make sense to create a `EnterprisePageWrapper` helper component that wraps `QueryClientProviderWrapper`, `AppContext.Provider`, etc.), perhaps accepts prop overrides/customizations, so it's a bit more DRY across test cases?